public/individualdevstats.php

```diff
@@ -1192,6 +1201,20 @@ function drawChart() {
             echo "</table></tbody>";
             echo "</br></br>";
 
+            /*
+             * Leaderboards
+             */
+            echo "<H1>Tickets</H1>";
```

Comment:
Is the intent to duplicate this H1?

Reply:
Fixed!

Reply:
Shouldn't it be just lowercase also (`<h1>`)?

Reply:
It seems to be mirroring the other H1 tags on the page, which is probably okay for the purposes of the PR but is a major semantic bug with the page itself as it should only have a single canonical H1. All the existing H1 tags are probably more well-suited as H2 tags.

Reply:
Ah, you're right. Better just leave it for some major cleanup later.